src/utils/consolidateStreamedStyles.js

```diff
@@ -21,6 +23,6 @@ export default function consolidateStreamedStyles() {
     }
 
     // $FlowFixMe
-    document.head.appendChild(frag)
```

Comment:
let's keep this code the way it is - there's no time we'd ever want the consolidated styles to not go to the end of `<head>`

Reply:
Yep, this will break the React rehydration as it would try to remove style tags and then add them back, right?